docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+db:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        #the values below should match the credentials used in 
+        #instance/local_settings.py
+        MYSQL_USER: nbsap
+        MYSQL_PASSWORD: nbsap
+        MYSQL_DATABASE: nbsap
+web:
+    build: .
+    command:
+        python instance/manage.py runserver 0.0.0.0:8000
+    volumes:
+        - .:/nbsap
+    ports:
+        - "8000:8000"
+    links:
+        - db
+    environment:
+        - DEBUG=True
```

Comment:
Please move this file to a different repository, see: https://taskman.eionet.europa.eu/projects/infrastructure/wiki/Docker_image_creation

I have created: https://github.com/eea/eea.docker.nbsap

This docker-compose.yml should be PR-ed there as a docker-compose.dev.yml. 

Another one with production setup (replicating demo, ec and country instances currently on calamar should be added).